modules/nixos/shared.nix

```diff
@@ -29,10 +29,10 @@
   nix.gc =
     {
       automatic = true;
-      options = lib.mkForce ''--max-freed "$((32* 1024**3 - 1024 * $(df -P -k /nix/store | tail -n 1 | ${pkgs.gawk}/bin/awk '{ print $4 }')))"'';
+      options = lib.mkForce ''--max-freed "$((48* 1024**3 - 1024 * $(df -P -k /nix/store | tail -n 1 | ${pkgs.gawk}/bin/awk '{ print $4 }')))"'';
     }
-    // lib.optionalAttrs pkgs.stdenv.isLinux (lib.mkForce {dates = "*:30";})
-    // lib.optionalAttrs pkgs.stdenv.isDarwin {interval.Minute = 30;};
+    // lib.optionalAttrs pkgs.stdenv.isLinux (lib.mkForce {dates = "*:17";})
```

Comment:
That will still run it only once per hour, or am I misunderstanding the format?

Reply:
good question. i'm somehow interpreting it as a frequency but that might not be true.

Reply:
I interpret it as a wildcard, so `*:*5` should run it every 10 minutes at minute 5, 15, 25 etc.

Reply:
just to point out the obvious but we only use one `*` i.e. `*:17`